### PR TITLE
WIP Bigger sync tests

### PR DESCRIPTION
**Pull request by karissa**
Added some new functions for writing a lot of hi res photos and records, which is more realistic for partners.

### CODE REVIEW

File: test/helpers.js [hunk 1]
```diff
@@ -35,3 +40,65 @@ function createApi (_, opts) {
...
+      pump(rs, ws, function (err) {
+        if (err) return cb(err)
+        var ws = mapeo.media.createWriteStream(`thumbnail/foo-${n}.jpg`)
+        var rs = fs.createReadStream(path.join(__dirname, 'hi-res.jpg'))
+        pump(rs, ws, function (err) {
+          if (err) return cb(err)
+          var ws = mapeo.media.createWriteStream(`original/foo-${n}.jpg`)
+          var rs = fs.createReadStream(path.join(__dirname, 'hi-res.jpg'))
+          pump(rs, ws, function (err) {
+            if (err) return cb()
```
Comment (noffle) [inline test/helpers.js]:
```suggestion
            if (err) return cb(err)
```

File: test/helpers.js [hunk 2]
```diff
@@ -35,3 +40,65 @@ function createApi (_, opts) {
...
+            toFinish++
+            console.log(`wrote ${toFinish}/${total}`)
```
Comment (noffle) [inline test/helpers.js]:
```suggestion
```

File: test/sync.js [hunk 3]
```diff
@@ -170,35 +170,19 @@ tape('sync: syncfile replication: osm-p2p-syncfile', function (t) {
...
   createApis(opts, function (api1, api2, close) {
     var pending = 4
+    var total = 200
 
     api1.sync.listen()
     api1.sync.on('target', written.bind(null, null))
     api2.sync.listen()
     api2.sync.on('target', written.bind(null, null))
-    writeBlob(api1, 'hello_world.png', written)
+    helpers.writeBigData(api1, total, written.bind(null, null))
```
Comment (noffle) [inline test/sync.js]: Why bind `null` to the `err` slot? It looks like `writeBigData` will return an error.

File: test/sync.js [hunk 4]
```diff
@@ -247,49 +233,38 @@ tape('sync: media: desktop <-> desktop', function (t) {
...
-    api1.sync.on('target', written.bind(null, null))
-    api2.sync.listen()
-    api2.sync.on('target', written.bind(null, null))
-    writeBlob(api1, 'hello_world.png', written)
-    writeBlob(api2, 'goodbye_world.png', written)
+    mobile.sync.listen()
+    mobile.sync.on('target', written.bind(null, null))
+    desktop.sync.listen()
+    desktop.sync.on('target', written.bind(null, null))
+    helpers.writeBigData(mobile, total, written.bind(null, null))
```
Comment (noffle) [inline test/sync.js]: same Q as above

File: test/sync.js (hunk 3 above)
Comment (karissa) [reply test/sync.js]: good point! was just a copy/paste

**Pull request merged**